modules/3d-tiles/src/lib/tileset/tileset-3d-traverser.js

```diff
@@ -314,7 +273,7 @@ export default class Tileset3DTraverser {
       let refines = false;
 
       if (this.canTraverse(tile)) {
-        refines = this.updateAndPushChildren(tile, stack, frameState) && parentRefines;
```

Comment:
@loshjawrence correct me if I am wrong, it seems not necessary to check `&& parentRefines `here.

Reply:
You want the parent refinment status to trickle down to the descendants. `refines` gets assigned to `tile._refines` at the end.